staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/pruning/algorithm.go

```diff
@@ -34,7 +54,12 @@ func Prune(obj interface{}, s *structuralschema.Structural, isResourceRoot bool)
 			s = &clone
 		}
 	}
-	prune(obj, s)
+	return prune(obj, s, opts)
+}
+
+// Prune calls into PruneWithOptions
+func Prune(obj interface{}, s *structuralschema.Structural, isResourceRoot bool) []string {
```

Comment:
drop the return value here if we're always setting ReturnPruned false via this invocation